uxarray/grid/coordinates.py

```diff
@@ -83,40 +88,75 @@ def node_xyz_to_lonlat_rad(node_coord):
     return [d_lon_rad, d_lat_rad]
 
 
-@njit(cache=ENABLE_JIT_CACHE)
-def normalize_in_place(node):
```

Comment:
There is another issue #346 where the requirement is `node` be a numpy array. It'd be good to tackle both together.  It has to be done carefully as it breaks numba compatibility.